
 
// console.log(addTwo(2)); // generates error
const addTwo = function(num){
    return num + 2;
}
console.log(addTwo(2));

console.log(addThree(3)); // runs perfectly
function addThree(num){
    return num + 3;
}